mlrun/datastore/sources.py

```diff
@@ -863,7 +861,7 @@ def __init__(
         **kwargs,
     ):
         """
-        Sets stream source for the flow. If stream doesn't exist it will create it
+        Sets stream source for the flow. If stream doesn't exist it will create it.
```

Comment:
Sets the stream source for the flow. If the stream doesn't exist it will create it.